nixos/modules/services/misc/etebase-server.nix

```diff
@@ -6,7 +6,9 @@ let
   cfg = config.services.etebase-server;
 
   pythonEnv = pkgs.python3.withPackages (ps: with ps;
-    [ etebase-server daphne ]);
+    [ etebase-server daphne ]
+    ++ optional (cfg.settings.database.engine == "django.db.backends.postgresql") psycopg2
+  );
```

Comment:
I think we can remove this since `withPostgres`'s default is `true`.
But I guess we could also set `withPostgres` to `false` by default and keep this.
Would be nice to have a third opinion on this.